services/docker-compose-dev.yml

```diff
@@ -20,6 +20,18 @@ services:
       - "../ida-ws/target:/app/ida-ws/target"
     networks:
       - ida-net
+  fuseki:
+    image: stain/jena-fuseki
+    ports:
+      - 3030:3030
+    environment:
+      - ADMIN_PASSWORD=pw123
```

Comment:
password in plain text

Reply:
Should be fixed now. The password can now be configured via the `FUSEKI_PW` file.

I still included a default password in the updated stack to make the development setup a bit easier. For a production deployment, the password can be reconfigured easily.

Reply:
Thanks @Cortys. Understandable!
@nikit91 can you also review and merge?